content/licensing.md

```diff
@@ -0,0 +1,18 @@
+# Licensing
+
+Licensing is about communicating what people are allowed to do with the things that you create (or what you are allowed to do with what other people create). This pertains to blogs, photographs, drawings, videos, designs, code, and research output (amongst many other things). Each piece of content has its own license. Licensing is one component to [Copyright](copyright.md).
+
+Licenses are a legal contract. As a creator, you can initiate legal steps if that contract is broken. However, the important question is whether you actually are committed to doing this before choosing a license. If you [transfer the copyright](copyright-transfer-agreements.md), the new rightsholder becomes responsible. If you grant an exclusive license, you become unable to grant other licenses without breaking that exclusive contract.
```

Comment:
There may be a note about the difference between copyright and author's right legislations. Under author's right legislation, only the diffusion, reproduction are ceased, not the m·p·aternity.

Reply:
Yea, author's rights (moral rights) and copyright are two different but closely intertwined things. Because it's not a module on the finesses of international law, I stuck  to this. Will try  to include this in the Copyright section.